doxygen/lang/145_basic_data_types.dox.tmpl

```diff
@@ -319,6 +357,44 @@ b += <abde77>;
     @par Type Name:
     \c "binary"
 
+    @anchor binary_slicing
+    @par Binary slicing:
+    Binary objects can be "sliced" by dereferencing them using the @ref list_element_operator "[] operator" with a range or a list, as in the following examples:\n
+    @code{.py}
+binary bin = <beadface>;
+
+# Creates a binary slice with the first two bytes of the original binary object
+binary bin1a = bin[0..1];
+binary bin1b = bin[0,1];
+binary bin1c = bin[..1];
+printf("binary 1a: %y\n", bin1a);
+printf("binary 1b: %y\n", bin1b);
+printf("binary 1c: %y\n", bin1c);
+
+# Creates a binary slice with the last two bytes of the original binary object in reverse order
+binary bin2a = bin[3..2];
+binary bin2b = bin[3,2];
+printf("binary 2a: %y\n", bin2a);
+printf("binary 2b: %y\n", bin2b);
+
+# Creates a binary slice with different bytes of the original binary object
+binary bin3 = bin[3,2..1,0];
+printf("binary 3: %y\n", bin3);
+
+# -------------------------
+# the output is as follows:
+
+binary 1a: !!binary vq0=
+binary 1b: !!binary vq0=
+binary 1c: !!binary vq0=
+binary 2a: !!binary zvo=
+binary 2b: !!binary zvo=
+binary 3: !!binary zvqtvg==
```

Comment:
It would be better here to convert the resulting binary into a hex string to better illustrate the output... From the `%y` output, which converts it to base64, it's not very clear.

Reply:
good suggestion - done